waspc/data/Generator/templates/server/src/config.js

```diff
@@ -1,5 +1,7 @@
 {{={= =}=}}
 import _ from 'lodash'
+{=! NOTE: validators are copied from shared templates =}
```

Comment:
I've went with the first approach so the resulting structure is:
```
scripts/
-- validate-env.mjs
-- shared/
---- validators.mjs
```

Reply:
Crap, didn't see this on time, I wanted to suggest option 2... sorry :sweat_smile: 

Shared stuff might contain more than just the the scripts stuff (e.g., types, application logic). We already have some template (and generated code) duplication with types this could help reduce (e.g., entity definitions).

Reply:
Scripts and src in my mind are separate scopes and I wanted to avoid cross-imports from one scope to another e.g. `import validators from '../src/shared/validators.mjs`

For example in the `scripts` scope we have a `shared` folder, but also in the `src` folder we have a `shared` folder. Does that make sense?

Reply:
Talked about this, we're using `shared` and `universal` to distinguish between the two top-level shared folders (user-defined and ours).

@infomiho Should we rename the user-defined one to `ext-shared`?

Reply:
@sodic That's a nice move, since we already have `ext-src` and use the `ext` to signal user-defined code. 

I'll make that change in this PR. Should we then still stick to `universal`? I'm on the fence tbh, it helps to further distinguish between `shared` and `ext-shared`, but then again, we do have `ext-src` and `src`.

Reply:
You can make the choice, I'd probably go with shared but wouldn't go out of my way to change it.

Reply:
We agreed that we'll keep `universal` for now, since going with `ext-shared` will require rewriting of paths for imports in the `client` and `server` dirs.

<img width="681" alt="Screenshot 2023-02-20 at 15 20 44" src="https://user-images.githubusercontent.com/2223680/220134407-b5ee04b1-ad22-4ae7-8f76-0fa32b2096f2.png">